### PR TITLE
	client: add GetVersion method

### PR DESCRIPTION
**Pull request by gyuho**
Cherry-picked from https://github.com/coreos/etcd/pull/6422.

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -477,6 +483,37 @@ func (c *httpClusterClient) AutoSync(ctx context.Context, interval time.Duration
 	}
 }
 
+type VersionResponse struct {
+	version.Versions
```
Comment (xiang90) [inline client/client.go]: how about just redefine version.Versions?

type VersionResponse version.Versions.

Comment (xiang90) [inline client/client.go]: or we really need the version response as a collection holder?

**Comment by gyuho** [comment]
@xiang90 Fixed. PTAL. Thanks.

**Comment by xiang90** [comment]
LGTM if CI passes.

**Pull request merged**